scripts/generator-adapter/generators/app/templates/src/endpoint/base.ts

```diff
@@ -0,0 +1,21 @@
+<% if (includeComments) { %>// Input parameters define the structure of the request expected by the endpoint.<% } %>
```

Comment:
I think these files could be stored with extensions like `*.ts.ejs`. This would allow VScode to do proper syntax highlighting, at least printing less errors

Reply:
Does it really work ? I'm using webstorm and changing to `.ts.ejs` actually makes it even worse. At least with `.ts` the colors and spacing for typescript code is correct, imports are navigating to correct paths. With  `.ts.ejs` for me everything looks like txt file.

Reply:
When naming a template with `*.ts`, this is what I see:
![image](https://github.com/smartcontractkit/ea-framework-js/assets/5432103/f8403845-c273-4823-90ad-66e7a619a55b)
I think regardless of whether there's support for ejs syntax, we should name the files with `.ejs` to better differentiate between template files and main src, that way we can easily filter and/or add rules for handling those within the IDE differently.

Reply:
Done